Automattic/jetpack
# Changes the Blaze Dashboard path to the new format

Some time ago (https://github.com/Automattic/wp-calypso/pull/79838), we updated the navigation path of Blaze to follow Calypso standards (we moved the site fragment to the end of the path). We currently have some compatibility redirections to make the change compatible with Jetpack.

This PR will update the paths to the Blaze Dashboard to the new path format.

Old Format: `tools.php?page=advertising#!/advertising/{SITE_SLUG}/posts/promote/post-{POST_ID}`
New Format: `tools.php?page=advertising#!/advertising/posts/promote/post-{POST_ID}/{SITE_SLUG}`

## Proposed changes:
* Updates the path used in the post row action and in the Post publish widget.
* Changes a typo in the Blaze configuration initial state. The Blaze Dashboard app is already compatible with both names (`intial_state` and `initial_state`).

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Go to the Post page and verify that the link generated for the "Promote with Blaze" link is correct (it has the site name at the end)
* Click the "Promote with Blaze" link for a post and verify that you land correctly in the Create campaign widget for the post you just selected
* Go to the Post page again, but now create a new post and publish it
* In the Post publish widget, click on the "Blaze this post" link
* Just as before, verify that you landed in the correct Blaze page for the published post.
